funnel/templates/forms.html.jinja2

```diff
@@ -250,6 +243,7 @@
 {%- endmacro %}
 
 {% macro tinymce3_editor(field, script=true) -%}
+  //Incase textarea uses tinymce, include `jquery.tinymce` in ext_requires in __init__.py
```

Comment:
This should be a Jinja2 comment using `{#- … #}`

Reply:
Also, the JS comment here is outside the JS `<script>` tag below.

Reply:
In fact we can remove this TinyMCE3 support entirely. It was used on Hasjob which has since upgraded to TinyMCE4. Version 3 was last used in Kharcha, which has been shutdown for several years.

Reply:
Removed